### PR TITLE
Update bootnodes

### PR DESCRIPTION
**Pull request by tomusdrw**

### CODE REVIEW

File: ethcore/res/ethereum/foundation.json [hunk 1]
```diff
@@ -173,9 +173,9 @@
 		"stateRoot": "0xd7f8974fb5ac78d9ac099b9ad5018bedc2ce0a72dad1827a1709da30580f0544"
 	},
 	"nodes": [
-		"enode://5a62f19d35c0da8b576c9414568c728d4744e6e9d436c0f9db27456400011414f515871f13a6b8e0468534b5116cfe765d7630f680f1707a38467940a9f62511@45.55.33.62:30303",
-		"enode://605e04a43b1156966b3a3b66b980c87b7f18522f7f712035f84576016be909a2798a438b2b17b1a8c58db314d88539a77419ca4be36148c086900fba487c9d39@188.166.255.12:30303",
-		"enode://dc72806c3aa8fda207c8c018aba8d6cf143728b3628b6ded8d5e8cdeb8aa05cbd53f710ecd014c9a8f0d1e98f2874bff8afb15a229202f510a9c0258d1f6d109@159.203.210.80:30303",
+		"enode://aafde2e81e035f417019a80f5342d1cd0e5bce97f83230fc57e1abbb3a9a5d6fb751446040c67261ed422324ffb69214567e181bb4ac0cc6e817451be0eaad1e@52.178.74.216:30303",
```
Comment (kirushik) [inline ethcore/res/ethereum/foundation.json]: Is this particular node online and available? I've run parity with `--reserved-only --reserved-peers` (with only three new enodes in the specified peers file) -- and only was able to connect to `460e` and `28fa`.

Comment (tomusdrw) [reply ethcore/res/ethereum/foundation.json]: @ddorgan ^^^ ?

Comment (andresilva) [reply ethcore/res/ethereum/foundation.json]: I was unable to connect to any of the 3 new nodes.

**Comment by 5chdn** [comment]
@paritytech/ci this is the highest priority right now, we are pushing a release.

File: ethcore/res/ethereum/foundation.json [hunk 2]
```diff
@@ -173,9 +173,11 @@
 		"stateRoot": "0xd7f8974fb5ac78d9ac099b9ad5018bedc2ce0a72dad1827a1709da30580f0544"
 	},
 	"nodes": [
-		"enode://5a62f19d35c0da8b576c9414568c728d4744e6e9d436c0f9db27456400011414f515871f13a6b8e0468534b5116cfe765d7630f680f1707a38467940a9f62511@45.55.33.62:30303",
-		"enode://605e04a43b1156966b3a3b66b980c87b7f18522f7f712035f84576016be909a2798a438b2b17b1a8c58db314d88539a77419ca4be36148c086900fba487c9d39@188.166.255.12:30303",
-		"enode://dc72806c3aa8fda207c8c018aba8d6cf143728b3628b6ded8d5e8cdeb8aa05cbd53f710ecd014c9a8f0d1e98f2874bff8afb15a229202f510a9c0258d1f6d109@159.203.210.80:30303",
```
Comment (5chdn) [inline ethcore/res/ethereum/foundation.json]: All three removed nodes are working for me: `5a62f`, `605e0`, `dc728`. Who is maintaining them?

File: ethcore/res/ethereum/foundation.json [hunk 3]
```diff
@@ -173,9 +173,11 @@
...
+		"enode://6a868ced2dec399c53f730261173638a93a40214cf299ccf4d42a76e3fa54701db410669e8006347a4b3a74fa090bb35af0320e4bc8d04cf5b7f582b1db285f5@163.172.131.191:30303",
+		"enode://66a483383882a518fcc59db6c017f9cd13c71261f13c8d7e67ed43adbbc82a932d88d2291f59be577e9425181fc08828dc916fdd053af935a9491edf9d6006ba@212.47.247.103:30303",
```
Comment (5chdn) [inline ethcore/res/ethereum/foundation.json]: Added nodes provided by @General-Beck `66a48` and `6a868` - both tested and working for me.

File: ethcore/res/ethereum/foundation.json [hunk 4]
```diff
@@ -173,9 +173,11 @@
...
+		"enode://aafde2e81e035f417019a80f5342d1cd0e5bce97f83230fc57e1abbb3a9a5d6fb751446040c67261ed422324ffb69214567e181bb4ac0cc6e817451be0eaad1e@52.178.74.216:30303",
+		"enode://460e54d7e9a361d326a9e503b3879c6a1075e1bfb7ea919b512ea1fe841e65f82c5f87af028f14a7825be1c1260825d5326b93b43a5bc72e3214a99e0c4c7bd4@52.230.6.166:30303",
+		"enode://28faaf6b2e86694d8978b8e6986e7813951d7bd25201116fa77de893aabedd2a4a8d5832776905b4c3e616320506516d08239d82aeef4355f6878c3a701a6059@40.71.19.172:30303",
```
Comment (5chdn) [inline ethcore/res/ethereum/foundation.json]: Can not connect to any of these three nodes: `aafde`, `460e5`, `28faa`.

**Comment by 5chdn** [comment]
Compare #7287

File: ethcore/res/ethereum/foundation.json (hunk 4 above)
Comment (ddorgan) [reply ethcore/res/ethereum/foundation.json]: Ran with these options:
parity --reserved-peers reserved  --reserved-only ui

Reserved peers file:
enode://5a62f19d35c0da8b576c9414568c728d4744e6e9d436c0f9db27456400011414f515871f13a6b8e0468534b5116cfe765d7630f680f1707a38467940a9f62511@45.55.33.62:30303
enode://605e04a43b1156966b3a3b66b980c87b7f18522f7f712035f84576016be909a2798a438b2b17b1a8c58db314d88539a77419ca4be36148c086900fba487c9d39@188.166.255.12:30303
enode://dc72806c3aa8fda207c8c018aba8d6cf143728b3628b6ded8d5e8cdeb8aa05cbd53f710ecd014c9a8f0d1e98f2874bff8afb15a229202f510a9c0258d1f6d109@159.203.210.80:30303

Connects to all three nodes and begins syncing:
2017-12-28 12:45:23 UTC Syncing  #182318 4a37…7da6   293 blk/s  330 tx/s   8 Mgas/s    117+    0 Qed   #182456    3/25 peers   2 MiB chain 68 MiB db 232 KiB queue 581 KiB sync  RPC:  1 conn, 13 req/s, 416 µs

Comment (tomusdrw) [reply ethcore/res/ethereum/foundation.json]: It shouldn't be syncing, it should be warp-syncing though.

**Pull request merged**